src/components/themed/SimpleTextInput.tsx

```diff
@@ -236,7 +251,7 @@ export const SimpleTextInput = React.forwardRef<SimpleTextInputRef, SimpleTextIn
           </TouchContainer>
         </InputContainerView>
       </EdgeTouchableWithoutFeedback>
-      {isIos && isFocused && (
+      {isIos && (isFocused || selected) && (
```

Comment:
`selected` is not always boolean (unless you take my suggestion to give it a default). So `selected === true` is lint 👌.